src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -106,20 +194,18 @@ private void updateOdometry() {
         );
     }
 
-    public void arcadeDrive(double linearSpeed, double rotationSpeed){
-        differentialDrive.arcadeDrive(linearSpeed, rotationSpeed);
-    }
-
+    @Override
     public void periodic() {
         updateOdometry();
+        updateShuffleboard();
 
         SmartDashboard.putNumber("angleOfBot", getAngleDegrees());
```

Comment:
why do we need all these SmartDashboard writes if you have the new shuffleboard entries approach?